packages/flutter_test/lib/src/controller.dart

```diff
@@ -189,7 +214,12 @@ class SemanticsController {
     final RenderView renderView = _controller.binding.renderViews.firstWhere((RenderView r) => r.flutterView == actualView);
 
     final List<SemanticsNode> traversal = <SemanticsNode>[];
-    _traverse(renderView.owner!.semanticsOwner!.rootSemanticsNode!, traversal);
+    _traverse(renderView.owner!.semanticsOwner!.rootSemanticsNode!, (SemanticsNode node) {
```

Comment:
The `simulatedAccessibilityTraversal` method this is in needs to get updated to use `FinderBase<SemanticsNode>`s.

Reply:
Updated.